tests/cases/fourslash/completionsDefaultKeywordWhenDefaultExportAvailable.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference path="fourslash.ts" />
+
+// https://github.com/microsoft/TypeScript/issues/3229
+
+// @filename: index.ts
+
+//// export default function () {}
+//// def/*1*/
+
+verify.completions(
+    { 
+        marker: "1", 
+        includes: [{ name: "default", kind: "keyword", sortText: completion.SortText.GlobalsOrKeywords }],
```

Comment:
I'd prefer to test this through smth like `exact: completion.globals` but I wasn't able to figure out quickly which one should I use here.